StaticAnalyzer/tests.py

```diff
@@ -52,18 +52,18 @@ def static_analysis_test():
         logger.info('Running PDF Generation Test')
         if platform.system() in ['Darwin', 'Linux']:
             pdfs = [
-                '/PDF/?md5=3a552566097a8de588b8184b059b0158&type=APK',
```

Comment:
I am personally a bit uncertain about these changes.  
Not entirely sure if this "PDF Generation Test" is something that should be changed.  
So please, add any feedback regarding this.

Reply:
Looks good to me. We want to run PDF generation tests for all kind of source formats we support.